Source/Android/jni/Host.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <mutex>
+
+#include "Core/Core.h"
+
+// The Core only supports using a single Host thread.
+// If multiple threads want to call host functions then they need to queue
+// sequentially for access.
+struct HostThreadLock
+{
+  static std::mutex s_host_identity_mutex;
```

Comment:
I *think* we use the `s_` prefix for stuff that is only visible in a single translation unit, and the `g_` prefix otherwise? This is practically a global at least, the `static` here doesn't actually affect linkage and is just C++ reusing the reserved keyword.

Reply:
I made it private in a later commit. Do you still think g_ makes the most sense in that situation? If so, I'll change it.

Reply:
Mhm, yeah I guess `s_` is fine if it's private.